Cargo.lock

```diff
@@ -246,9 +246,9 @@ dependencies = [
 
 [[package]]
 name = "async-std"
-version = "1.13.0"
+version = "1.13.1"
```

Comment:
@Byron there was conflicts in the lockfile, I ended up deleting it and recreating it using `cargo update` and that bumps a lot of dependencies (which I think is a good thing to do periodically)